staff/lgimeneza/css-layout-simple-adaptive/styles/main.css

```diff
@@ -0,0 +1,27 @@
+@import url('mobile.css') only screen and (max-width: 768px);
+@import url('desktop.css') only screen and (min-width: 768px);
+
+*{
+    background: black;
+    color: white;
+    border: 1px solid white;
+    box-sizing: border-box;
+    padding: 0px;
+    margin: 0px;
+    text-align: center;
+}
+html{
+    height: 100%;
+}
+body{
+    height: 100%;
+}
+header{
+    height: 20%;
+}
+footer{
+    height: 20%;
+}
```

Comment:
I have to complain...you know why ;P you could save extra space writting together header & footer... and in mobile.css, nav & aside